src/main/java/ca/uhn/fhir/jpa/starter/service/HelperService.java

```diff
@@ -151,31 +147,105 @@ public void refreshMapForOrgId(String orgId) {
 		FhirClientProvider fhirClientProvider = new FhirClientProviderImpl((GenericClient) FhirClientAuthenticatorService.getFhirClient());
 		mapOfOrgHierarchy.put(orgId, ReportGeneratorFactory.INSTANCE.reportGenerator().getOrganizationHierarchy(fhirClientProvider, orgId));
 	}
-	
-	
-	
+
+	private String getKeycloakGroupId(String uid) {
```

Comment:
This parameter can be renamed to `groupName` or something similar. This `uid` is nothing but the group name in keyclock. Correct me if I am wrong.